landoapi/hg.py

```diff
@@ -109,6 +114,10 @@ class HgRepo:
         "extensions.strip": "",
         "extensions.rebase": "",
         "extensions.set_landing_system": "/app/hgext/set_landing_system.py",
+
+        # Turn on fix extension for autoformatting, set to abort on failure
+        "extensions.fix": "",
+        "fix.failure": "abort",
```

Comment:
We should probably put this in `config_overrides` per-repo, since most repos will not have this extension enabled.

Reply:
Enabling this extension doesn't do anything except turn on the `hg fix` command, which is the same behaviour as enabling `purge`/`rebase`/`strip`, and we enable those globally. I think there are more reasons to turn this on globally than to leave it off (violates DRY, prone to error if someone forgets to set error to abort, makes formatting configs more verbose than they need to be, etc).